### ISSUE TITLE
Hammock boat

### ISSUE THREAD

**Issue by LoganVYBE** [GitaVYBE/VYBE-pm#47]

**Comment by GitaVYBE** [comment]
What is this?

**Comment by LoganVYBE** [comment]
Idea for an EDDIY episode 😅

**Comment by GitaVYBE** [comment]
@LoganVYBE can you please post some details on your idea here?

**Comment by LoganVYBE** [comment]
Yeah it can be backlogged, but Eli is gonna build a hammock like they sell at costco and we're going to attempt to engineer it into a boat
